resources/lang/nl/coreui.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+return [
+
+    'default_title'               => 'CoreUI Laravel Theme door HZ-HBO-ICT',
+    'settings'                    => 'Instellingen',
+    'full_name'                   => 'Volledige naam',
+    'email'                       => 'E-mailadres',
+    'password'                    => 'Wachtwoord',
+    'retype_password'             => 'Wachtwoord nogmaals invoeren',
+    'remember_me'                 => 'Ingelogd blijven',
+    'register'                    => 'Registreren',
+    'register_a_new_membership'   => 'Registreer een nieuw account',
+    'i_forgot_my_password'        => 'Ik ben mijn wachtwoord vergeten',
+    'i_already_have_a_membership' => 'Ik heb al een account',
+    'sign_in'                     => 'Inloggen',
+    'log_out'                     => 'Uitloggen',
+    'toggle_navigation'           => 'Schakel navigatie',
+    'login_message'               => 'Log in om je sessie te starten',
+    'register_message'            => 'Registreer een nieuw account',
+    'password_reset_message'      => 'Wachtwoord herstellen',
+    'reset_password'              => 'Wachtwoord herstellen',
+    'send_password_reset_link'    => 'Stuur email met link voor wachtwoordherstel',
+    'enter_email_message'         => 'Vul uw email addres in',
```

Comment:
Typo: "Vul uw emailadres in"

Reply:
https://onzetaal.nl/taaladvies/website-e-mailadres-web-site-e-mail-adres/